src/main/java/mekanism/client/ClientRegistration.java

```diff
@@ -577,6 +581,16 @@ public static void registerItemColorHandlers(RegisterColorHandlersEvent.Item eve
         }
     }
 
+    @SubscribeEvent
+    public static void registerItemDecorations(RegisterItemDecorationsEvent event) {
+        var mekaSuitDecorator = new ChemicalFluidBarDecorator(true,
+                itemStack -> itemStack.getItem() instanceof ItemMekaSuitArmor armor && //only show bar for parts that have the module installed
+                        (armor.hasModule(itemStack, MekanismModules.NUTRITIONAL_INJECTION_UNIT) || armor.hasModule(itemStack, MekanismModules.JETPACK_UNIT)),
```

Comment:
Rather than `hasModule` we may want to consider doing `isModuleEnabled`, though I need to think about what makes more sense (for now you can probably leave this as hasModule)

Reply:
it's the check that the fluid/gas handler use so I just used them for now